packages/client/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -1,3 +1,20 @@
+import { Card, Typography } from 'antd'
+
+import { LoginForm } from '../../components/LoginForm'
+import React from 'react'
+import style from './LoginPage.module.scss'
+
+const { Title } = Typography
+
 export const LoginPage = () => {
-  return <div>Страница входа</div>
+  return (
+    <React.Fragment>
+      <Card className={style['login-page']}>
+        <Title level={2} className={style['login-title']}>
+          Login
```

Comment:
```suggestion
          Login
```

Стоит использовать кириллицу для единообразия.

Reply:
согласен. поменял.